modules/flowable-engine/src/test/java/org/flowable/engine/test/api/task/DelegateTaskTest.java

```diff
@@ -68,7 +69,12 @@ public void testChangeCategoryInDelegateTask() {
             assertEquals("approval", task.getCategory());
             Map<String, Object> taskVariables = new HashMap<>();
             taskVariables.put("outcome", "approve");
-            taskService.complete(task.getId(), taskVariables, true);
+
```

Comment:
FWIW, this is based on an older version of the file as line 68/69 is now:
<pre>
  assertThat(task.getCategory()).isEqualTo("approval");
</pre>
but it is still possible to merge your changes.  There is at least on other such file in this PR.

Reply:
test class updated